tests/e2e/test_launch.py

```
@@ -105,6 +105,30 @@ def test_package_import():
+            "-c",
+            "from src.chains.slide_gen_chain import SlideGenChain; "
+            "print('Chain imports successful')",
+        ],
+        cwd=test_helper.project_root,
+        capture_output=True,
+        text=True,
+        timeout=10,
+    )
+
+    if result.returncode != 0:
+        assert False, f"Chain import failed:\n{result.stderr}"
```

Ruff (0.12.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)